pkg/omf/cli/omf.install_package.fish

```diff
@@ -1,5 +1,11 @@
-function omf.install_package
+function omf.install_package -d "Return success for at least one successful install."
+  set -l return_code 1
   for search in $argv
     omf.install --pkg $search
+    set -l code $status
+    if test $code -eq 0
```

Comment:
Maybe do the same you did on remove and only install one package at a time by adding the loop to the function caller?